app/views/layouts/_links.html.erb

```diff
@@ -10,21 +10,50 @@
     </a>
   </header>
   <div class="card-content" data-card-target="content" hidden="true">
+    <div class="media">
+      <div class="media-left">
+        <span class="icon is-large">
+          <i class="fas fa-globe fa-3x"></i>
+        </span>
+      </div>
+      <div class="media-content">
+        <p class="title is-4">Vanjske poveznice</p>
+        <p class="subtitle is-6">Provjerene poveznice na korisne dokumente i web stranice</p>
+      </div>
+    </div>
     <div class="content">
-      <ul>
-        <li><a href="https://docs.google.com/document/d/15kSw0O1jtZYTeLh3jZg2XeGFbNbavIBsA0u_PvK8YRs/mobilebasic"  target="_blank">
-          Pomoć obiteljima iz Petrinje, Siska i okolice - kontakti, smještaj i prijevoz - Google doc (preview)
-        </a></li>
-        <li><a href="https://docs.google.com/document/d/1yUxHtKCkkXEQ8tXTD2bODbisxwF5Q9zvPJoU27vQGUM/edit?fbclid=IwAR0g1F__7avcVpOlf__NQgBq626LDmDNyz42k545TT5MCjTLMvf4De9tvMs" target="_blank">
-          Pomoć za Petrinju - Google doc
-        </a></li>
-        <li><a href="https://www.facebook.com/groups/155412969344431/permalink/155812429304485/" target="_blank">
-          POMOC GLINI, PETRINJI, SISKU i okolnim mjestima - Facebook grupa
-        </a></li>
-        <li><a href="https://potres2020.openit.hr/views/map" target="_blank">
+      <div class="is-full my-1">
```

Comment:
Why did you remove the `<ul>` ?

Reply:
List was adding bullets and it somehow seemed easier to remove it and add divs. But in can revert it to `<ul>` and add scss changes so that bullets are not added.

Reply:
Yes, please do.

Reply:
Solved. Please check!